AirCasting/CreateSessionViews/ChooseSessionTypeView.swift

```diff
@@ -16,6 +16,7 @@ struct ChooseSessionTypeView: View {
     @State private var isMobileLinkActive = false
     @State private var didTapFixedSession = false
     @EnvironmentObject var bluetoothManager: BluetoothManager
+    let urlProvider: BaseURLProvider
```

Comment:
can this be `private`?

Reply:
Same as above, preview is screaming at me :/